### DISCUSSION TITLE
Local Update Set Slash Command

### DISCUSSION THREAD

**Discussion by CapnSammeh** [arnoudkooi/SN-Utils#345] [Q&A]
Hey team!

I often end up needing to quickly refer to an Update Set that someone on the team is working in - I don't necessarily need to _switch_ into the Update Set, but being being able to quickly review the records they're working on is invaluable. 

Interested in your feedback on this one!

**Comment by arnoudkooi** [comment]
Could you try adding the below custom Slashcommand?
This will allow you to search by name or application.
It limits the search to Update sets that have an Update Record in the last 30 days.

URL:
`nameLIKE$0^ORapplication.nameLIKE$0^RLQUERYsys_update_xml.update_set,>=1^sys_updated_onONLast 30 days@javascript:gs.beginningOfLast30Days()@javascript:gs.endOfLast30Days()^ENDRLQUERY^ORDERBYDESCsys_updated_on`

Fields (suggested)
`name,application`

![Google Chrome20221221 - 2356](https://user-images.githubusercontent.com/840781/209018913-7f4e6212-34fe-4a13-a26e-c9c1fbc3495d.png)

**Comment by arnoudkooi** [comment]
I'm only seeing now you added a PR for this, although I appreciate the involvement,
the custom commands are more appropriate for this, rather than having it baked in.

I'll be looking at a way to make it easier to share slash commands next year.

Thanks!

**Comment by r-coh** [comment]
Yo, @arnoudkooi  and @CapnSammeh 

why am i getting spammed with NSFW content... is some1 misappropriating this project?? 

<img width="1317" alt="Screenshot 2023-01-03 at 11 37 54" src="https://user-images.githubusercontent.com/38593081/210323762-d7459d1a-b181-4778-9608-c3380ab7633a.png">

**Reply by arnoudkooi** [reply]
Apparently, spam found its way to GitHub :(. I'm removing and reporting these messages when they occur, that's all I can do...
